src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -329,6 +330,44 @@ begin
            ennreal.tendsto_coe, ennreal.to_nnreal_coe],
 end
 
+lemma tendsto_lintegral_nn_filter_of_le_const {ι : Type*} {L : filter ι} [L.is_countably_generated]
```

Comment:
can you add docstrings to this lemma and the next ones? When the assumptions are several lines long, docstrings are often easier to parse (you can skip irrelevant details in the docstring, not so much in proper Lean code :-)